src/cli/paths/query.yml

```diff
@@ -0,0 +1,81 @@
+post:
```

Comment:
I had to make a separate `query.yml` for the CLI codegen to use. Nothing new is added here, but the possible input/output types have been cut down to only the things used by the current CLI (flux input, CSV output).